### PR TITLE
Rollbar reporter that scrubs private data

### PR DESCRIPTION
**Pull request by brianberlin**
#### Description: 
By default Rollbax uses Rollbax.Reporter.Standard. This is a copy of that with modifications to scrub keys for things like passwords and api keys.

#### Reviewer don't-forgets:

- [ ] Test coverage feels appropriate, given potential risk
- [ ] We're not doubling down on already-bad code
- [ ] If there are web UI changes, they don't add anything that could be considered client-side page navigation (unless pre-approved as being necessary by another engineer)
- [ ] If there are web UI changes, they don't add any AJAX form submits (unless pre-approved as being necessary by another engineer)
- [ ] If there are any lint rules disabled, they are disabled per-line, and were (in the reviewer's judgment) appropriate to disable
- [ ] Any new environment variables used in the app are both documented and have been added to both staging and production environments already
- [ ] Potential race conditions are either inconsequential, or the code prevents them from occurring.
- [ ] If this is a UI change that called for screenshots/GIFs, in the reviewer's judgement, they were included

### CODE REVIEW

**Comment by bryanjos** [comment]
There may be some overlap between this and the error reporter here: https://github.com/revelrylabs/revelry_phoenix_app_template/blob/master/lib/app_template_web/error_reporter.ex

I really do like the approach of adding the rollbax reporter here. There may be usefullness in having both. I would make sure that the list of keys to ignore or filter are the same between the 2.

**Comment by oohnoitz** [comment]
One thing I've encountered is the credentials being inside of the `stacktrace` itself and not the state. This was a bit difficult to figure out how to sanitize properly.

File: lib/app_template/rollbax_reporter.ex [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  defp handle_error_format('** State machine ' ++ _ = message, data) do
+    if charlist_contains?(message, 'Callback mode') do
+      :next
+    else
+      handle_gen_fsm_error(data)
+    end
+  end
+
+  # Errors in a regular process.
+  defp handle_error_format('Error in process ' ++ _, [pid, {reason, stacktrace}]) do
```
Comment (grossvogel) [inline lib/app_template/rollbax_reporter.ex:83]: Total sidebar... I just noticed these are all charlists instead of binaries... anybody know why?

Comment (brianberlin) [reply lib/app_template/rollbax_reporter.ex:83]: I do not know why. I noticed that as well and went to double check that it wasn't a mistake on my part. https://github.com/ForzaElixir/rollbax/blob/v0.11.0/lib/rollbax/reporter/standard.ex#L1

Comment (oohnoitz) [reply lib/app_template/rollbax_reporter.ex:83]: I think this is the equiv of `"Error in process " <> _` and has to do with the way a bitstring ends up getting expanded for erlang?

**Pull request merged**